package.json

```diff
@@ -11,11 +11,11 @@
     "type-check": "tsc --noEmit",
     "analyze": "ANALYZE=true yarn build",
     "test": "jest",
-    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
-    "lint:fix": "yarn lint --fix",
+    "eslint": "eslint . --ext .js,.jsx,.ts,.tsx",
+    "eslint:fix": "eslint . --ext .js,.jsx,.ts,.tsx --fix",
     "prettier": "prettier --ignore-path .gitignore \"./**/*.{js,jsx,ts,tsx}\"",
```

Comment:
I'm not sure why we have 2 prettier commands? It seems you don't want prettier to run on certain files for some reason?

I've renamed the `format` commands to `prettier` commands, as we use Prettier to format the files. Same reasoning for the `lint` -> `eslint` rename.

Reply:
I'll go revert my changes to the format and lint commands. 😉